common/partition/default-partitioner.go

```diff
@@ -36,8 +36,8 @@ import (
 )
 
 const (
-	defaultPartitionConfigWorkerIsolationGroup = "worker-isolation-group"
-	defaultPartitionConfigRunID                = "wf-run-id"
+	IsolationGroupKey = "isolation-group"
+	WorkflowRunIDKey  = "wf-run-id"
```

Comment:
Why don't we use workflow ID instead of run ID? Workflow ID would give a more consistent experience across resets, continueAsNews, etc., which would make everything easier to reason about from user's perspective.

Reply:
Fair, Let me follow that up in a different diff